lib/containers/docker.pm

```diff
@@ -26,9 +26,7 @@ sub configure_insecure_registries {
     my $registry = registry_url();
```

Comment:
Is this `registry` variable no more used?

Reply:
yes, I've seen it in some places, I didn't want to touch it.